src/Symfony/Bridge/Twig/composer.json

```diff
@@ -22,7 +22,7 @@
     "require-dev": {
         "symfony/phpunit-bridge": "~2.7",
         "symfony/finder": "~2.3",
-        "symfony/form": "~2.6",
+        "symfony/form": "~2.6,>=2.6.6",
```

Comment:
I agree. We can think about it again once Composer releases alpha10 (see composer/composer#3779).